### DISCUSSION TITLE
Let me in

### DISCUSSION THREAD

**Discussion by Mike-FreeAI** [community/community#135264] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I need to try it

**Comment by Akash1134** [comment]
Hi @Mike-FreeAI 

Sign up for the limited public beta [HERE](https://gh.io/models).

Thanks!

**Reply by Mike-FreeAI** [reply]
I did

**Comment by Mike-FreeAI** [comment]
I did
